extensions/wrapper/wrapper-common-api/src/main/java/de/sovity/edc/ext/wrapper/api/common/model/UiAsset.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Copyright (c) 2022 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.ext.wrapper.api.common.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@RequiredArgsConstructor
+@Schema(description = "Type-Safe Asset Metadata as needed by our UI")
+public class UiAsset {
+    @Schema(description = "Asset ID", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String assetId;
+
+    @Schema(description = "Asset name")
+    private String name;
```

Comment:
to be merged with UIAsset from other PR?

Reply:
Yes, it will be, but we should use this PR as base as soon as it works.